src/compute/src/communication.rs

```diff
@@ -8,210 +8,300 @@
 // by the Apache License, Version 2.0.
 
 //! Code to spin up communication mesh for a compute replica.
-//! Most of this logic was taken from Timely's communication::networking
-//! module, with the crucial difference that we
-//! actually verify that we can receive data on a connection before considering it
-//! established, and retry otherwise. This allows us to cope with the phenomenon
-//! of spuriously accepted connections that has been observed in Kubernetes with linkerd.
+//!
+//! The startup protocol is as follows:
+//! The compute controller in `environmentd`, after having connected to all the
+//! `computed` processes in a replica, sends each of them a `CreateTimely` command
+//! containing an epoch number (which is the same across all copies of the command).
+//! The meaning of this number is irrelevant to us,
+//! as long as it increases monotonically.
```

Comment:
> as long as it increases monotonically
nit: can you clarify the scope of this invariant (for example, should it be preserved across `environmentd` restarts or is it sufficient to have a monotonic sequence per `environmentd` process lifetime)?